src/services/clientFacade.ts

```diff
@@ -0,0 +1,239 @@
+import { Client, reponseMeta, SearchDocFormData } from "../types";
+
+
+interface ClientSearchResponse {
+    data: Client[];
+    meta: reponseMeta;
+}
+  
+
+export default class ClientFacade {
+    private apiUrl: string;
+  
+    constructor(apiUrl: string) {
+        this.apiUrl = apiUrl;
+    }
+
```

Comment:
rename files from clientFacade.ts to cliente-facade.ts.
do the same with other files.